### PR TITLE
Saml troubleshooting doc improvement 

### PR DESCRIPTION
**Pull request by DaedalusG**
This is just to surface a bit about how to get SAML response info for folks since it seems to be common that Admins don't know how to get this info.

### CODE REVIEW

File: doc/admin/auth/saml/index.md [hunk 1]
```diff
@@ -72,8 +74,29 @@ Then, confirm there are no error messages in:
 The most likely error message indicating a problem is `Error prefetching SAML service provider metadata`. See [SAML troubleshooting](#troubleshooting) for more tips.
 
 ## Troubleshooting
+Security Assertion Markup Language (SAML) is a common web protocol used pass authorized credentials between two web applications, a service provider (SP, Sourcegraph in this instance), and an Identity Provider (IdP, see our [list](#IdentityProviders) of IdP's above). This communication is conducted via XML assertions.
```
Comment (bobheadxi) [inline doc/admin/auth/saml/index.md:77]: Nice context! Maybe this should just be at the very top of the doc?

File: doc/admin/auth/saml/index.md [hunk 2]
```diff
@@ -72,8 +74,29 @@ Then, confirm there are no error messages in:
...
+1. Naviagate in your browser to Sourcegraph, prepare to attempt a login via SAML in Sourcegraph
+2. Open your developer tools and navigate to the Network tab. Check to see if theres an option to preserve logs, if so enable it.
+3. Clear the collection of network logs in the your devTool network tab and attempt a saml login
+4. Look for a network request in the network tab that indicates a SAML request response communication. (This might be labeled ACS, or Authn), select the network request and observe its headers. You should see something like the image below from a Sourcegraph Okta login, observed via Safari devTools (don't worry this example has no sensative info)
+
+![Screen Shot 2021-09-15 at 1 13 17 PM](https://user-images.githubusercontent.com/13024338/134255811-88250622-7f0e-42f8-91b0-a3f7bf5274fc.png)
+
+
+
+
```
Comment (bobheadxi) [inline doc/admin/auth/saml/index.md:96]:
```suggestion

```

File: doc/admin/auth/saml/index.md [hunk 3]
```diff
@@ -72,8 +74,29 @@ Then, confirm there are no error messages in:
...
 
+### Enable logging in Sourcegraph containers
 Set the env var `INSECURE_SAML_LOG_TRACES=1` to log all SAML requests and responses on:
 
 - [Docker Compose](../../install/docker-compose/index.md) and [Kubernetes](../../install/kubernetes/index.md): the `sourcegraph-frontend` deployment
 - [Single-container](../../install/docker/index.md): the `sourcegraph/server` container
+
+### Debugging with your browser
+When debugging a problem with SAML its often helpful to use your browsers developer tools to directly observer the XML assertions and their contents. Below are some general pointers on how to collect SAML communications:
+1. Naviagate in your browser to Sourcegraph, prepare to attempt a login via SAML in Sourcegraph
```
Comment (bobheadxi) [inline doc/admin/auth/saml/index.md:87]:
```suggestion
When debugging a problem with SAML its often helpful to use your browsers developer tools to directly observer the XML assertions and their contents. Below are some general pointers on how to collect SAML communications:

1. Naviagate in your browser to Sourcegraph, prepare to attempt a login via SAML in Sourcegraph
```

**Review by bobheadxi — COMMENTED** [review]
Nice! Some formatting nits and a suggested move :)

File: doc/admin/auth/saml/index.md (hunk 1 above)
Comment (DaedalusG) [reply doc/admin/auth/saml/index.md:77]: Yeah that makes sense

**Review by bobheadxi — APPROVED** [review]
Nice! 🚀

**Review by rhiam — COMMENTED** [review]
This is really amazing @DaedalusG, it will surely help with my jumpCloud reproduction

**Pull request merged**